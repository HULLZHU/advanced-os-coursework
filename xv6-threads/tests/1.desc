simple test of kernel thread library
