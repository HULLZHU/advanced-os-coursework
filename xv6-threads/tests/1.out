XV6_TEST_OUTPUT : global = 12345 i = 314159
